apps/web/src/components/Composer/Actions/ReferenceSettings/index.tsx

```diff
@@ -31,24 +49,135 @@ const ReferenceSettings: FC = () => {
   const isMyFollowers = isFollowerOnlyReferenceModule && onlyFollowers;
   const isMyFollows = isDegreesOfSeparationReferenceModule && degreesOfSeparation === 1;
   const isFriendsOfFriends = isDegreesOfSeparationReferenceModule && degreesOfSeparation === 2;
+  const [followValue, setfollowValue] = useState(
+    isMyFollows
+      ? MY_FOLLOWS.value
+      : isMyFollowers
+      ? MY_FOLLOWERS.value
+      : isFriendsOfFriends
+      ? FRIENDS_OF_FRIENDS.value
+      : isEveryone
+      ? EVERYONE.value
+      : EVERYONE.value
+  );
 
   interface ModuleProps {
     title: string;
+    description: string;
     icon: ReactNode;
-    onClick: () => void;
-    selected: boolean;
   }
```

Comment:
Move props before the component starts

`const Module: FC<ModuleProps> = ({ title, icon, onClick, selected }) => (`